sdk/python/feast/infra/offline_stores/bigquery_source.py

```diff
@@ -291,3 +295,28 @@ def to_proto(self) -> SavedDatasetStorageProto:
 
     def to_data_source(self) -> DataSource:
         return BigQuerySource(table=self.bigquery_options.table)
+
+
+class BigqueryLoggingDestination(LoggingDestination):
+    _proto_attr_name = "bigquery_destination"
+
+    table_ref: str
```

Comment:
should probably be consistent with our other names (this would be table instead of table_ref)

Reply:
similar for other DWH